src/sdk/vscode/README.md

```diff
@@ -39,3 +39,11 @@ All requirements will be checked on each command invocation
 - `iotea.rule.or` + `Strg + Space` > Create new Or Rule
 - `iotea.rule.op` + `Strg + Space` > Create a new Rule with an Operation Constraint
 - `iotea.rule.change` + `Strg + Space` > Create a new Rule with a Change Constraint
+
+## Troubleshooting
+
+- _"Docker version check fails"_<br>
+  Make sure you have the latest Docker version installed, since `docker version --format=json` is used to retrieve the version. In older versions the string "json" was returned
```

Comment:
This way of checking the docker version _only_ works on windows, and from what I can tell, only works because of a bug in the newly introduced wrapper around com.docker.cli, which when invoked directly behaves according to the documentation and identically on all platforms.

The proper way to retrieve the version according to the [docker documentation](https://docs.docker.com/engine/reference/commandline/version/) is:

`docker version --format='{{.Client.Version}}'`

which works on all platforms tested.

Reply:
Or more bluntly:
In it's current form, it's not possible to use the VSCode extension on linux without manually patching the source and recompile the extension (which isn't very user friendly).

Reply:
In 0.9.3 we will use `docker version --format={{.Client.Version}}`